c2corg_ui/static/js/outingediting.js

```diff
@@ -96,7 +95,7 @@ app.OutingEditingController.prototype.successRead = function(response) {
   // check if user has right to edit -> the user is one of the associated users
   if (this.auth.hasEditRights(outing['associations']['users'])) {
     outing = this.formatOuting_(outing);
-    this.differentDates = app.utils.areDifferentDates(outing['date_start'], outing['date_end']);
+    this.differentDates = (window.moment(outing['date_start']).diff(outing['date_end']) === 0) ? false : true;
```

Comment:
if !! means a double negation, and `window.moment(outing['date_start']).diff(outing['date_end']);` == `!!window.moment(outing['date_start']).diff(outing['date_end']);`, what's the deal?